serving/src/main/java/ai/djl/serving/models/ModelManager.java

```diff
@@ -416,6 +417,18 @@ public CompletableFuture<Map<String, Object>> workerStatus() {
                                         if (wlm.getWorkerPool(wpc).isFullyScaled()) {
                                             data.put(modelName, new StatusResponse("Healthy"));
                                         } else {
+                                            boolean sageMakerHealthCheckOverride =
+                                                    Boolean.parseBoolean(
+                                                            Utils.getenv(
+                                                                    "SAGEMAKER_HEALTH_CHECK_OVERRIDE",
+                                                                    "false"));
+                                            if (sageMakerHealthCheckOverride) {
+                                                logger.warn(
```

Comment:
This should be info level

Reply:
updated, and also changed the other log we had for SERVING_HEALTH_CHECK_OVERRIDE from error -> info